_config.yml

```diff
@@ -0,0 +1,12 @@
+# Site settings
+title: Citiparks Cleaning Contract
+email: pittsburgh@codeforamerica.org
+description: > # this means to ignore newlines until "baseurl:"
```

Comment:
Do you want to update this? I'm not sure if it's used anywhere, but it is available in the `{{ site.description }}`, so you might want to do that.